internal/wire/parse.go

```diff
@@ -856,6 +859,13 @@ func allFields(call *ast.CallExpr) bool {
 	return strings.EqualFold(strconv.Quote("*"), b.Value)
 }
 
+// isPrevented checks whether field i is prevented by tag "-".
+// Since this is the only tag used by wire, we can do string comparison
+// without using reflect.
+func isPrevented(st *types.Struct, i int) bool {
+	return strings.Contains(st.Tag(i), `wire:"-"`)
```

Comment:
This isn't technically following the same algorithm that the reflect package uses for splitting tags. However, I would be surprised if this actually causes issues in practice, since struct tag usage is fairly minimal. Perhaps add a TODO to use the more precise parsing algorithm later?

Reply:
Filed #179